hal/src/nRF52840/sleep_hal.cpp

```diff
@@ -844,14 +981,6 @@ static int enterHibernateMode(const hal_sleep_config_t* config, hal_wakeup_sourc
         wakeupSource = wakeupSource->next;
     }
 
-    // Disable PWM
-    nrf_pwm_disable(NRF_PWM0);
```

Comment:
Do we not need this?

Reply:
We've called `hal_pwm_sleep(true, nullptr);` before getting here.

Reply:
:+1: